qa/repository-multi-version/build.gradle

```diff
@@ -92,7 +92,7 @@ configurations {
   testArtifacts.extendsFrom testImplementation
 }
 
-task testJar(type: Jar) {
+def testJar = tasks.register("testJar", Jar) {
```

Comment:
`TaskProvider`

Reply:
you mean we should used typed variables here? testJar is of type TaskProvider